pennylane/ftqc/ftqc_device.toml

```diff
@@ -0,0 +1,82 @@
+schema = 3
```

Comment:
The toml files are tidy in the sense that they tie into something we've already set up with the device API for defining how things work, but we only need a few things from them, and those things could just be hard-coded in the prototype instead of retrieved from `capabilities` based on a `toml` file. I'm very open to just deleting these if we feel it keeps the prototype simpler. Thoughts?

Reply:
I see value in both paths. I think we are ok with not being too "tidy" for now if it helps move things along. We can always delete these later

Reply:
Alright, I'll leave as-is for now!